docs/en/operations/system-tables.md

```diff
@@ -933,6 +933,79 @@ query_id:      acc4d61f-5bd1-4a3e-bc91-2180be37c915
 trace:         [94222141367858,94222152240175,94222152325351,94222152329944,94222152330796,94222151449980,94222144088167,94222151682763,94222144088167,94222151682763,94222144088167,94222144058283,94222144059248,94222091840750,94222091842302,94222091831228,94222189631488,140509950166747,140509942945935]
 ```
 
+## system.stack_trace {#system-tables_stack_trace}
+
+Contains stack traces of all server threads. Allows developers to introspect the server state.
+
+To analyze logs, use the `addressToLine`, `addressToSymbol` and `demangle` [introspection functions](../sql-reference/functions/introspection.md).
+
+Columns:
+
+-   `thread_id` ([UInt64](../sql-reference/data-types/int-uint.md)) — Thread identifier.
+-   `query_id` ([String](../sql-reference/data-types/string.md)) — Query identifier.
```

Comment:
Look the the `query_id` description in the [trace_log](https://clickhouse.tech/docs/en/operations/system-tables/trace_log/) docs.